.travis.yml

```diff
@@ -76,6 +76,9 @@ jobs:
     - name: "Symfony 4.x"
       php: 7.2
       env: DEPENDENCIES="dunglas/symfony-lock:^4"
+    - name: "Symfony 5.x"
+      php: 7.2
+      env: DEPENDENCIES="dunglas/symfony-lock:^5"
```

Comment:
This does not exist.

Reply:
I added the native symfony env variable to lock dependencies.